assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.js

```diff
@@ -438,6 +439,7 @@ ConversionReportingNotificationCTAWidget.propTypes = {
 	WidgetNull: PropTypes.elementType,
 };
 
-export default whenActive( { moduleName: 'analytics-4' } )(
-	ConversionReportingNotificationCTAWidget
-);
+export default compose(
+	whenActive( { moduleName: 'analytics-4' } ),
+	whenHasChangedConversionEvents()
+)( ConversionReportingNotificationCTAWidget );
```

Comment:
This is a very big component with a lot of hooks, so an HOC is the only clean way to preserve this condition.